### ISSUE TITLE
Prevent screen from sleeping when recording or playing audio

### ISSUE THREAD

**Issue by someoneEsle** [wireapp/wire-android#256]
I very often lose messages I'm recording just because the screen turns off :'(
While we're at it you may also prevent the screen from sleeping while I'm playing incoming audio messages

**Comment by someoneEsle** [comment]
Tested after building from master, works as expected now

**Comment by someoneEsle** [comment]
I actually have to reopen this issue. It works fine most of the time, but I have found a bug that I was able to reproduce successfully.
1. Record an audio message A, and send it.
2. While audio A is being sent, start recording another audio message B (in the same conversation).
3. If you don't touch the screen, at some point it turns off while you're recording.

I tested this with Anna. Here is the debug log: 
[wire_debug_report.txt](https://github.com/wireapp/wire-android/files/593422/wire_debug_report.txt)
